src/OrbitQt/CallTreeWidget.cpp

```diff
@@ -184,6 +185,74 @@ void CallTreeWidget::ResizeColumnsIfNecessary() {
   column_resizing_state_ = ColumnResizingState::kDone;
 }
 
+static int FindDepthDifferenceFromIndexToDeepestVisibleNode(QTreeView* tree_view,
```

Comment:
Nit: Too long name, how about? `FindHeightOfVisibleNodesSubTree `

Reply:
Long names are not a problem (E.g., style guide: "Do not worry about saving horizontal space as it is far more important to make your code immediately understandable by a new reader."). However, I like expressing the concept like that: I changed to `ComputeHeightOfSubtreeOfVisibleNodes`.

Reply:
Yeah, I wanted to point out to Long names that can be nicer being short. I like yours!